src/setup/txt_joybinput.c

```diff
@@ -98,6 +99,9 @@ static void CanonicalizeButtons(void)
     int vbutton;
     int i;
 
+    static_assert(arrlen(all_joystick_buttons) == NUM_VIRTUAL_BUTTONS,
```

Comment:
Next time someone forgets to increase NUM_VIRTUAL_BUTTONS, it will print a warning while chocolate-setup will be compiling. I'll submit the changes soon. 
```
txt_joybinput.c:59:5: warning: excess elements in array initializer
     &joybmenu,
     ^
txt_joybinput.c:59:5: note: (near initialization for ‘all_joystick_buttons’)
txt_joybinput.c:60:5: warning: excess elements in array initializer
     &joybautomap,
     ^
txt_joybinput.c:60:5: note: (near initialization for ‘all_joystick_buttons’)
```